packages/application-components/src/theming.ts

```diff
@@ -12,54 +12,6 @@ const newShadow1 = '0px 2px 5px 0px rgba(0, 0, 0, 0.15)';
 
 export const themesOverrides = {
   default: {
-    colorForPageHeaderBottomBorder: uiKitDesignTokens.colorNeutral,
```

Comment:
`test` theme has been removed and all `default` theme values has been overwritten with the ones we had in the `test` one.